iris-logo/latexmkrc

```diff
@@ -0,0 +1,4 @@
+$latex = 'latex %O %S && dvisvgm --no-font %D';
```

Comment:
@bordaigorl so do you recommend we add `--optimize=all` here (which seems to be equivalent to `--optimize`)?

Reply:
yes `--optimize` and optionally `--zoom=5`